validator/validator-main.protoascii

```diff
@@ -4871,6 +4871,14 @@ tags: {
     name: "crossorigin"
     value: "anonymous"
   }
+  attrs: {
+    name: "crossorigin"
+    value: "anonymous"
+  }
+  attrs: {
+    name: "crossorigin"
+    value: "anonymous"
+  }
```

Comment:
These look like some kind of merge was applied backwards. There shouldn't be multiple of these same attrs. Same below.

Reply:
Thanks for catching this. Fixed it.